### PR TITLE
Error email subject

### PR DESCRIPTION
**Pull request by dnil**
This PR adds a functionality or fixes a bug.

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. `ssh your.email@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
</details>

**How to test**:
1. arrange a crash

**Expected outcome**:
The functionality should be working. Email header should contain hostname.
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by CR
- [x] tests executed by DN

### CODE REVIEW

**Comment by dnil** [comment]
Seems I still have at least one instance of last weeks `black` installed somewhere... Until I find my error, I will blame PyCharm for hiding it.

**Comment by dnil** [comment]
I was partly right; PyCharm was using a different one, but it was configurable, so I guess that also shifts the blame to me. 😄

**Comment by northwestwitch** [comment]
💯 

image

File: scout/server/app.py [hunk 1]
```diff
@@ -238,11 +238,12 @@ def configure_email_logging(app):
 
     from scout.log import TlsSMTPHandler
 
+    mongodbname = app.config["MONGO_DBNAME"]
     mail_handler = TlsSMTPHandler(
         mailhost=app.config["MAIL_SERVER"],
         fromaddr=app.config["MAIL_USERNAME"],
         toaddrs=app.config["ADMINS"],
-        subject="O_ops... {} failed!".format(app.name),
+        subject="O_ops... {} failed on db {}!".format(app.name, mongodbname),
```
Comment (northwestwitch) [inline scout/server/app.py:246]: Smart!

**Review by northwestwitch — APPROVED** [review]
Works perfectly! 👍🏻 

Let's deploy soonish so we can use to track down errors from the new servers! 
(At least if we deploy on stage and we get an error with unknown database. we'd know that it comes from prod..)

**Pull request merged**